lib/helpers/MediaApiHelper.ts

```diff
@@ -188,4 +188,31 @@ export class MediaApiHelper {
 
     return await this.create(media);
   }
+  
+  async createDefaultMediaWithImage(mediaName: string, parentId?: string) {
+    const mediaType = await this.api.mediaType.getByName('Image');
+    const crypto = require('crypto');
+    const temporaryFileId = crypto.randomUUID();
+    const fileName = 'Umbraco.png';
+    const filePath = './fixtures/mediaLibrary/' + fileName;
+    const mimeType = 'image/png';
+    await this.api.temporaryFile.create(temporaryFileId, fileName, mimeType, filePath);
+    
+    const media = new MediaBuilder()
+      .withMediaTypeId(mediaType.id)
+      .addVariant()
+        .withName(mediaName)
+        .done()
+      .addValue()
+        .withAlias('umbracoFile')
+        .withValue(temporaryFileId)
+        .done()
+      .build();
+
+    if (parentId !== undefined) {
```

Comment:
It would be better if you had another function called 'createDefaultMediaWithImageAndParentId' to avoid pushing the value directly.